logging/advancedlog/main.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2019 Google LLC
```

Comment:
```suggestion
// Copyright 2020 Google LLC
```

Reply:
Updated. Should this be updated every time I edit an old file as well? Or it's timestamped as of the date of file creation?